src/main/java/core/basesyntax/colorsupplier/ColorSupplier.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax.colorsupplier;
+
+import core.basesyntax.Figure;
+import core.basesyntax.figures.Circle;
+import java.util.Random;
+
+public class ColorSupplier {
+
+    public ColorSupplier() {
+    }
+
+    public static Figure getDefaultFigure() {
```

Comment:
Don’t use static methods in your solution - see [common mistakes](https://mate-academy.github.io/jv-program-common-mistakes/java-core/abstract-class-interface/oop-advanced#dont-use-static-methods-in-your-solution)